doc/src/ldd_indicator_dpd.rst

```diff
@@ -0,0 +1,43 @@
+.. index:: pair_coeff ldd indicator dpd
```

Comment:
Having all these different ldd_*_*.rst files as separate files and with indexing etc seems like overkill. 

Please have a look at pair style granular (both implementation and documentation).
It has a somewhat similar architecture where you have classes derived from `GranSubMod` that implement different model variants that can be selected via keywords. Those are all documented in the page for pair style granular. I think something similar would be more suitable here and make the documentation more compact and accessible (instead of being scattered).